src/main/java/com/manardenza/utils/DatabaseSeeder.java

```diff
@@ -1,6 +1,58 @@
 package com.manardenza.utils;
 
+import com.manardenza.entity.Hotel;
+import com.manardenza.entity.Room;
+import com.manardenza.entity.User;
+import lombok.Getter;
+
+import java.util.ArrayList;
+import java.util.List;
+
 public class DatabaseSeeder {
 
-    // Class to fill DAO implementations
+    @Getter private static List<User> usersList = new ArrayList<>();
+    @Getter private static List<Room> roomsList = new ArrayList<>();
+    @Getter private static List<Hotel> hotelsList = new ArrayList<>();
+
+    static {
+        usersList.add(new User(464655, "Vasya", "Petrov"));
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "Petrov" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1192)